06-instructions.md

```diff
@@ -3,12 +3,12 @@
   Created: 2021-02-03
 </pre>
 
-# 06. Instructions & State Digests
+# 06. Instructions
 
 ## Overview
 
 This RFC specifies the messages exchanged between the user's swap client and its own daemon.
-As sketched elsewhere, the `client`→`daemon` route consists of (a) `instructions` from the client to daemon that control the state transition of an ongoing swap, and (b) the `daemon`→`client` route consists of `state digests` sent to the client encoding the `daemon`'s swap state tailored specific to client's control and presentation functionality. The `client` must  present control choices to the end-user during the progrssion of the protocol execution.
+As sketched elsewhere, the `client`→`daemon` route consists of (a) `instructions` from the client to daemon that control the state transitions of an ongoing swap, and (b) the `daemon`→`client` route consists of `state digests` sent to the client encoding the `daemon`'s swap state tailored specific to client's control and presentation functionality. The `client` must  present control choices to the end-user during the progression of the protocol execution.
```

Comment:
Do we still mention the daemon->client state digest here?

If we do then a link to `09` might be good no?